src/Utility/Test/TestThreadGroups.C

```diff
@@ -62,6 +62,18 @@ TEST(TestThreadGroups, Groups2D)
   CHECK_INTMATRICES_EQUAL(groups[0], "src/Utility/Test/refdata/ThreadGroups_2D_1.ref");
 #endif
 
+  std::vector<bool> b14(4, true);
+  std::vector<bool> b24(4, true);
+
+  std::string ref1 = "src/Utility/Test/refdata/ThreadGroups_2D_strip1_p1.ref";
+  std::string ref2 = "src/Utility/Test/refdata/ThreadGroups_2D_strip1_p2.ref";
```

Comment:
As you may know, I would rather inline those small reference arrays instead to read from file.
Anyway, two of them (the strip2 data) seem not to be used at all.